UnityProject/Assets/Scripts/HealthV2/Living/Damage/BodyPartDamage.cs

```diff
@@ -8,18 +8,19 @@ namespace HealthV2
 	public partial class BodyPart
 	{
 		/// <summary>
-		/// The amount damage taken by this is modified by
+		/// The armor of the clothing covering a part of the body, ignoring selfArmor.
 		/// </summary>
-		[Tooltip("The amount that damage taken by this is modified by")]
-		public Armor BodyPartArmour = new Armor();
+		[Tooltip("The armor of the clothing covering a part of the body, ignoring selfArmor.")]
+		public readonly LinkedList<Armor> ClothingArmors = new LinkedList<Armor>();
 
 		/// <summary>
+		/// The armor of the body part itself, ignoring the clothing (for example the xenomorph's exoskeleton).
 		/// The amount damage taken by body parts contained within this body part is modified by
 		/// increases as this body part sustains more damage, to a maximum of 100% damage when the
 		/// health of this part is 0.
 		/// </summary>
-		[Tooltip("The amount that damage taken by contained body parts is modified by")]
-		public Armor SubOrganBodyPartArmour = new Armor();
```

Comment:
There is technically 3 Armors , clothing, body part itself and then protection of sub body parts

Reply:
so,  this is so you can implement Gollum's that take less damage from certain sources inherently,
 and the sub body armour is for stuff like a skull that would protect the organs below it

Reply:
`SelfArmor` and `SubOrganBodyPartArmour` are united. You can still make Gollum, give his skull `SelfArmor`, and then other body parts.

Reply:
but you can't separate how much damage is done to the body part itself versus how much damage is transferred to the sub- organs?

Reply:
I can by assigning different armor values to the body part and its innards since the organs are also body part

Reply:
I think I understood you, I see the problem now. I'll fix it

Reply:
> I think I understood you, I see the problem now. I'll fix it.

Or wait, no 😄 . I don't need to set organ's armor, but I use `selfArmor` of the body part to protect its organs.

Reply:
Done.

(on the third try I finally understood what you meant. I feel stoopid)

Reply:
looks good